playbooks/common/openshift-cluster/initialize_openshift_version.yml

```diff
@@ -0,0 +1,29 @@
+---
+# NOTE: requires openshift_facts be run
+- name: Determine openshift_version to configure on first master
```

Comment:
Granted that is something that can be addressed as a follow-up, since we are still struggling to break backwards compat for 1.9.4 currently.